pos_product_sync/models/product_product.py

```diff
@@ -54,3 +62,18 @@ def change_product_qty(self):
             product_ids.append(wizard.product_id.id )
         self.env['product.product'].send_field_updates(product_ids)
         return res
+
+
+class PosOrder(models.Model):
+
+    _inherit = "pos.order"
+
+    # Compatibility with pos_product_available
+    def _process_order(self, pos_order):
+        res = super(PosOrder, self)._process_order(pos_order)
+        order = self.create(self._order_fields(pos_order))
```

Comment:
You duplicate the ``create_from_ui method`` functionality and destroy accounting as well. Do not create a new order, you can find all necessary data from the ``pos_order`` dictionary.

Reply:
Done